### ISSUE TITLE
consumer didn't commit message to Kafka.

### ISSUE THREAD

**Issue by NineMvp** [Jroland/kafka-net#55]
I try by example code https://github.com/Jroland/kafka-net   with kafka version 0.8.2.1. 

First time, I run the consumer to consume all message in queue.
but the second time the consumer start to consume message offset 0 again. 

I found your code in unit test has something like a commit the message but do not work

```csharp
public bool Commit(Message message)
        {
            using (var routerx = new BrokerRouter(Options))
            {
                var conn = routerx.SelectBrokerRoute(this.Topic, message.Meta.PartitionId);

                var commit = CreateOffsetCommitRequest(ConsumerGroup, message.Meta.PartitionId, message.Meta.Offset);

                var response = conn.Connection.SendAsync(commit).Result.FirstOrDefault();

                
                if (response != null && response.Error == 0)
                    return true;
                else
                    throw new ApplicationException(string.Format("Cannot commit message of Topic:{0}, Group:{1}, Partition:{2}, Offset:{3}", Topic, ConsumerGroup, message.Meta.PartitionId, message.Meta.Offset));
                    
            }
        }

```

please help,

**Comment by NineMvp** [comment]
I fixed by add local commit log 

thanks,

**Comment by Jroland** [comment]
Hey @NineMvp There is a PR that fixes the commits for version 8.2.1 of kafka.  Right now the nuget package only works with 8.1.  If you look at the integration branch, you will see the fixed merged in there.  I have not been able to test the change yet, which is why I have not pushed it up to the nuget package.

**Comment by shinurag** [comment]
> I fixed by add local commit log
> 
> thanks,

Can you please elaborate this, as am facing the same issue. using latest version of kafka:- kafka_2.13-2.8.0
